proto/cosmos/base/store/v1beta1/snapshot.proto

```diff
@@ -10,7 +10,9 @@ message SnapshotItem {
   // item is the specific type of snapshot item.
   oneof item {
     SnapshotStoreItem store = 1;
-    SnapshotIAVLItem  iavl  = 2 [(gogoproto.customname) = "IAVL"];
+    SnapshotIAVLItem  iavl = 2 [(gogoproto.customname) = "IAVL"];
+    SnapshotKVItem    kv = 3 [(gogoproto.customname) = "KV"];
+    SnapshotSchema schema = 4;
```

Comment:
In https://github.com/cosmos/cosmos-sdk/pull/10961/files it's moved to a new package. 
Lets invite @yihuang  for the next call to discuss how we should converge

Reply:
One thing that comes to my mind is the state-sync hooks PR should be backward compatible and could be backported to older versions. So it's better to merge that first, then do the breaking changes on top of that.